src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/sort/FindAndRerankSort.java

```diff
@@ -49,12 +59,13 @@ public DataRecorder recordTo(DataRecorder dataRecorder) {
   public boolean equals(Object obj) {
     return Objects.equals(vectorizeSort, ((FindAndRerankSort) obj).vectorizeSort)
```

Comment:
nit: should probably check that `obj` is a `FindAndRerankSort` first, to avoid `ClassCastException` and having to cast on every line. So like:

```
    return (obj instanceof FindAndRerankSort other)
        && Objects.equals(vectorizeSort, other.vectorizeSort)
        && Objects.equals(lexicalSort, other.lexicalSort)
        ...
```